ktor-network/posix/src/io/ktor/network/util/SocketAddress.kt

```diff
@@ -5,7 +5,10 @@
 package io.ktor.network.util
 
 import kotlinx.cinterop.*
+import platform.darwin.*
```

Comment:
Import is needed for `inet_ntop` which is not in posix. I think there would need to be separate implementations for Darwin, Linux, and Windows. Could I create these implementations using expect/actual or is there a better way?

Reply:
Please consider creating required methods in `cinterop` network module. It can use `#ifdef` to write platform-specific code

Reply:
Alright, that would work. See: https://github.com/ktorio/ktor/blob/56d25383602d33cc1b792776fe4374f7572418f9/ktor-network/posix/interop/network.def

I am not familiar with the Windows C APIs, could you help me here, please?